spec/spec_helper.rb

```diff
@@ -11,6 +11,7 @@
 require "rspec/rails"
 require "support/active_record"
 require "rspec/its"
+require "pry"
```

Comment:
Was this intentional?

Reply:
And I made a separate commit to do this (24195ad45e5653c3b92f7b84bf050f6e6477aa29) - just feels like nice thing to have in specs.